frontend/stepper/python/index.ts

```diff
@@ -109,29 +110,26 @@ export default function(bundle: Bundle) {
             const source = state.buffers.source.model.document.toString();
 
             if (platform === 'python') {
-                const context = {
-                    infos: {},
-                    aceEditor: null,
-                    onError: (diagnostics) => {
-                        const response = {diagnostics};
-
+                const context = getRunningContext();
```

Comment:
Context needs to be kept between executions, as long as we're on the same task level, and reset before each new execution.

Background executions will also need to be supported at some point (not necessarily in this pull request though, but good to keep in mind).